pkg/kubectl/resource_printer.go

```diff
@@ -600,7 +606,7 @@ func printEndpointsList(list *api.EndpointsList, w io.Writer, withNamespace bool
 }
 
 func printNamespace(item *api.Namespace, w io.Writer, withNamespace bool, columnLabels []string) error {
-	if _, err := fmt.Fprintf(w, "%s\t%s\t%s\n", item.Name, formatLabels(item.Labels), item.Status.Phase); err != nil {
+	if _, err := fmt.Fprintf(w, "%s\t%s\t%s\t%s\n", item.Name, formatLabels(item.Labels), item.Status.Phase, translateTimestamp(item.CreationTimestamp)); err != nil {
```

Comment:
Can you please delete the last `\n`? It shouldn't be there from the beginning. Thanks